test/gtest/common/gtest-all.cc

```diff
@@ -7921,7 +7921,7 @@ const char kCurrentDirectoryString[] = ".\\";
 # endif  // GTEST_OS_WINDOWS_MOBILE
 #else
 const char kPathSeparator = '/';
-const char kPathSeparatorString[] = "/";
+//const char kPathSeparatorString[] = "/";
```

Comment:
Should this be removed instead of commented out?

Reply:
removed
thank you